tests/e2e/tests/admin/login.spec.ts

```diff
@@ -14,26 +14,27 @@ test.describe('Login', () => {
     test('A user should be able to log in with or without making their authentication persistent', async ({
       page,
       context,
+      browser,
     }) => {
       // Test without making user authentication persistent
       await login({ page });
-      await expect(page).toHaveTitle('Homepage | Strapi');
+      await expect(page).toHaveTitle(TITLE_HOME);
 
+      // Close the page and open a new one
       await page.close();
-
-      page = await context.newPage();
-      await page.goto('/admin');
-      await expect(page).toHaveTitle('Strapi Admin');
+      const nonPersistentPage = await context.newPage();
```

Comment:
Re-using the page object was rarely but sometimes causing a "page already closed" when attempting to navigate after